src/libmongoc/doc/mongoc_server_api_version_from_string.rst

```diff
@@ -0,0 +1,26 @@
+:man_page: mongoc_server_api_version_from_string
+
+mongoc_server_api_version_from_string()
+=======================================
+
+Synopsis
+--------
+
+.. code-block:: c
+
+  bool
+  mongoc_server_api_version_from_string (const char *version,
+                                         mongoc_server_api_version_t *out);
+
+Given a string ``version``, populates ``out`` with the equivalent :symbol:`mongoc_server_api_version_t` if ``version`` represents a valid API version.
+
+Paremeters
+----------
+
+* ``version``: A string representing the version number.
```

Comment:
I would avoid using the term "version number" here and in other documentation. The version is not required to be a number, it just so happens that the first one is "1". There may be other versions in the future that are not identifier by a number.

Reply:
Oh that's really interesting!  So maybe we'll start naming MDB versions after desserts??  Ha.  I will change this to be "version identifier" instead of "version number".

Reply:
Desserts are already taken by Android, but we could always do something else, like cocktails for example 🍹

Reply:
`MONGOC_VERSION_DARK_AND_STORMY`